README.md

```diff
@@ -220,6 +224,7 @@ Available options:
 * ``array`` - Enter multiple values. POST and PUT page only.
 * ``file`` - A file-input form element, to upload files as `Content-Type: multipart/form-data`. All non-file form inputs will be sent as individual string values. The current implementation supports only one file input per form.
 * ``password`` - A password text box
+* ``note`` - A plain text note within the other fields. Use ``note`` property for text. ``label`` is optional.
```

Comment:
Just to make sure, it's just a simple text to place in the POST/PUT editor?

WDYT about adding a "tooltip" as an extra property to each option. This will add a tooltip next to each field's label.

Reply:
This puts text directly into the edit page. This is what it looks like:
![image](https://user-images.githubusercontent.com/7407231/57403829-63321780-718f-11e9-99c1-f2dca6153f41.png)

For my use case I need something that's a bit more prominent than a tooltip, but I would love to have that option for some other fields.

Reply:
Cool.